src/messages_manager.ts

```diff
@@ -54,10 +58,16 @@ export class MessagesManager implements Disposable {
                         }
                     }
                 }
+
                 this.channel.show(true);
-                this.channel.append(str);
+                this.writeMessage(str);
                 break;
             }
         }
     }
+
+    private writeMessage(msg: string) {
+        logger.info(msg);
```

Comment:
Do we need to add it to the log?

Reply:
I waffled on this. My take is we should, as if we're ever asking a user for logs to diagnose a problem, these messages may be indicative of actions they took

Reply:
It also resolves the concern about preserving timestamps

Reply:
OK